### DISCUSSION TITLE
How to run this with kaggle?

### DISCUSSION THREAD

**Discussion by Mertsudo** [erew123/alltalk_tts#459] [Q&A]
Trying to run on kaggle instead colab because kaggle gives 2x gpu can you help me everytime trying to run cell just kicks me out.

**Comment by erew123** [comment]
Hi @Mertsudo Unfortunately my work load with AllTalk and general life is huge at the moment, so I dont have time to go look at things like running on alternative platforms as that can be very in depth and complicated. If there is specific errors/code youre welcome to post it and I can attempt to give pointers, but I dont have time currently to setup accounts, test scenarios, debug etc as this can be a 10-20 hours of my time (based on previous experience with google colab).
